src/index.ts

```diff
@@ -18,12 +18,12 @@ export const run = () => {
       }
     });
 
-    if (!args[0] || !AVAILABLE_COMMANDS.includes(args[0])) {
-      showHelp(args[0], options.help);
+    if (!args[0] || !AVAILABLE_COMMANDS.includes(args[0]) || args.length > 2) {
+      showHelp(args, options.help);
     } else if (args[0] === 'android') {
-      const androidSetup = new AndroidSetup(options);
+      const androidSetup = new AndroidSetup(options, args[1]);
```

Comment:
`AndroidSetup` class is an exposed API from the tool (it can be imported and used in any other project), so we should not make changes to this API unless absolutely necessary, as it might impact other projects as well (`nightwatch` being one of them).

Instead, we can either handle the sub-commands in this file only or maybe pass the subcommand as a separate option in the `options` object.